apps/server/src/modules/files-storage/files-storage-api.module.ts

```diff
@@ -1,15 +1,15 @@
+import { HttpModule } from '@nestjs/axios';
 import { Module } from '@nestjs/common';
 import { CoreModule } from '@src/core';
-import { LoggerModule } from '@src/core/logger';
 import { AuthModule } from '@src/modules/authentication';
 import { AuthorizationModule } from '@src/modules/authorization';
-import { FileSecurityController } from './controller/file-security.controller';
-import { FilesStorageController } from './controller/files-storage.controller';
+import { FileSecurityController, FilesStorageController } from './controller';
 import { FilesStorageModule } from './files-storage.module';
+import { FileRecordUC, FilesStorageUC } from './uc';
 
 @Module({
-	imports: [AuthorizationModule, FilesStorageModule, AuthModule, CoreModule, LoggerModule],
+	imports: [AuthorizationModule, FilesStorageModule, AuthModule, CoreModule, HttpModule],
```

Comment:
HttpModule is needed? Do we inject in on a place in files-storage module? Otherwise it should not needed.

Reply:
It is used in uploadFromUrl to get the file. So I gues it's needed here.